torch/nn/utils/meta_init.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I don't think this should be in `torch/nn/utils` - shouldn't it be somewhere distributed-specific?

Reply:
Well I believe there is still not a consensus on that. For instance I know that @dzhulgakov wants to see this work under `torch.nn` as he thinks that this would be useful beyond distributed use cases. On the other hand I know that several core team members are really not keen to see this in the core package. Right now I will publish the remaining PRs as they are. Once the work is properly reviewed, we can decide where to land it at the final stage. Would that work for you?